src/agent/Makefile

```diff
@@ -32,6 +32,16 @@ EXTRA_RUSTFEATURES :=
 ##VAR SECCOMP=yes|no define if agent enables seccomp feature
 SECCOMP := yes
 
+# If LIBSECCOMP_LINK_TYPE or LIBSECCOMP_LIB_PATH is not set,
+# we assume that libseccomp is not installed in the environment.
+# Therefore, the agent disables seccomp feature.
```

Comment:
If you agree with my previous comment, then that logic would obviously have to be changed ;-)

Reply:
I removed these changes.